src/applications/simple-forms/21-0966/containers/IntroductionPage.jsx

```diff
@@ -112,7 +154,7 @@ class IntroductionPage extends React.Component {
                 </p>
               </va-alert>
               <p className="vads-u-margin-top--3">
-                If you don’t want to verify your identity right now, you can
+                If you don't want to verify your identity right now, you can
```

Comment:
@jeana-adhoc Did you mean to change all of these apostrophes? It looks like you only changed them in the introduction page, so I wasn't sure if that was intentional or not

Reply:
It was actually - as I thought curly quotes were bad - but I later realized there's a reason. I'll revert those quotes.

Reply:
Quote have been updated.. new commit added